libraries/AP_Param/AP_Param.cpp

```diff
@@ -1947,6 +1948,17 @@ void AP_Param::convert_old_parameters(const struct ConversionInfo *conversion_ta
     flush();
```

Comment:
You don't need contents of the method anymore if your calling into the other one.